it/src/test/java/org/corfudb/universe/scenario/ConcurrentClusterResizeIT.java

```diff
@@ -34,9 +34,16 @@ public void concurrentClusterResizeTest() {
         // Deploy a five nodes cluster
         final int numNodes = 5;
 
-        getScenario(numNodes).describe((fixture, testCase) -> {
-            ClientParams clientFixture = fixture.getClient();
-            CorfuCluster corfuCluster = universe.getGroup(fixture.getCorfuCluster().getName());
+        workflow(wf -> {
+            wf.setupDocker(fixture -> fixture.getCluster().numNodes(numNodes));
+            wf.setupProcess(fixture -> fixture.getCluster().numNodes(numNodes));
+            wf.setupVm(fixture -> fixture.getCluster().numNodes(numNodes));
```

Comment:
I don't get this. Why are we setting up all three?

Reply:
1. In `UniverseManager` you define what type of the universe you would like to have.
2. in the test you don't know what type of the universe is going to be used
3. we define how many nodes we want to have for all possible types of the universe. In general, there could be different settings for different types of universes. The fixtures of the different universes can have completely different fields and settings, so you have to define a parameter like `numNodes` for all universes in this case.

And yes, we can provide common settings across all universes, like `numNodes`, but I wouldn't like to do that now. For now, it's not clear what settings I'd like to make  common across different universes